### DISCUSSION TITLE
zwave locking up wifi NIC?

### DISCUSSION THREAD

**Discussion by no1warr1or** [zwave-js/zwave-js#4278] [General]
Before I walk into this topic, I know these devices communicate on different bands. Even more so the device I'm discussing is using the 5Ghz band. This is not a serious problem, i have a solution already. More so find it interesting and would like to dig more and discuss 

That being said, I have a zwave device (GE 40amp outdoor switch) that is setup in HA and has a schedule to shut off my pool pump at 6PM.

I have a dell micro desktop used for camera live viewing, currently over wifi 5Ghz (yes I know wifi bad lol hardwire is planned when I get a new 42u Rack and redo wiring).

What's interesting is every night at 6pm when the automation triggers, at the same time the wifi card in this desktop goes out, and I mean to the point where I have to restart the computer. 

Interesting things I've noted is that I get a ton of RX/TX drops in zwave to this node specifically, and when the wifi drops on the desktop it reports an absurd wifi frequency, I have a screenshot somewhere I'll try to attach later. 

Anyone experienced or seen anything like this before? Is it even possible? Or just coincidence.

**Comment by AlCalzone** [comment]
The only thing I can imagine is a power issue (where the Z-Wave USB stick and the Wifi card aren't getting enough voltage) or some interference is going on that's strong enough to throw the wifi card off (not necessarily on the Wifi bands).

To be sure - is your Z-Wave stick placed away from everything else, as described [here](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/connectivity-issues?id=general-troubleshooting)?

**Reply by no1warr1or** [reply]
The Zstick I have is in my server, the PC mentioned above is a separate system on the other side of the wall, somewhat close ill admit.

As far as what that article recommends I have it on a USB extension leading outside of my rack, aside from the GE switch mentioned my other 8 nodes have 0 issues even in scheduled tasks. And that GE node still functions as normal. 

What kind of interference could do that? Im also wondering if the pool pump is on the same 120v leg that the PC is branched off of. I guess I have some investigation to do.

**Reply by AlCalzone** [reply]
> aside from the GE switch mentioned my other 8 nodes have 0 issues

But do you know the issue is not caused by bad controller placement?

**Reply by no1warr1or** [reply]
To be honest, I don't know for sure

**Reply by AlCalzone** [reply]
Wouldn't hurt trying - the success rates of using a USB extension is unbelievably high.

**Reply by no1warr1or** [reply]
I have a USB extension that makes the stick point up currently sitting on top of my rack already so I'll have to play around with locations and see what happens.
